src/js/views/replyTweet.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Tweet from './tweet.js';
+import ReplyListener from './replyListener.js';
```

Comment:
import {ReplyListener} from './replyListebner.js' 
this may make it a singleton, read about it